### DISCUSSION TITLE
in version 12, #is appended multiple times for dynamic routes

### DISCUSSION THREAD

**Discussion by tamilarasue1995** [vercel/next.js#44948] [Help]
when entering http://localhost:3000/dynamic#hash?q=12, this url is automatically modified into http://localhost:3000/dynamic#hash#hash#hash#hash#hash#hash#hash#hash#hash?q=12

The url should not be modified...
Even though this issue is fixed in nextjs 13, is there a way to fix this in next v12 itself, like a patch fix?

**Comment by icyJoseph** [comment]
What version of next 12 is doing this?

**Reply by tamilarasue1995** [reply]
"next": "12.2.0"

**Reply by icyJoseph** [reply]
Ah I missed one thing, what does dynamic do? is it a catch all route? or dynamic parameters?
